tox.ini

```diff
@@ -1,36 +1,52 @@
+[base]
+pkg_name = ansible_navigator
+mypy_tmp_dir = mypy_tmp_dir
+
 [tox]
+envlist = testenv, linters, type, report, clean
 minversion = 1.4.2
-envlist = linters
-skipsdist = True
 skip_missing_interpreters = true
+skipsdist = True
 
 [testenv]
+allowlist_externals = cat
+                      rm
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/test-requirements.txt
 commands =
-    py.test -v
+    py.test -v --cov {[base]pkg_name} --cov share --cov-report term-missing --cov-branch
 
-[testenv:black]
-install_command = pip install {opts} {packages}
-commands =
-  black -v {toxinidir}
+[testenv:clean]
+deps = coverage
+skip_install = true
+commands = coverage erase
+           rm -rf ./{[base]mypy_tmp_dir}
 
 [testenv:linters]
 install_command = pip install {opts} {packages}
```

Comment:
Just curious...not changed in this PR, but why is this line necessary? Shouldn't it default to something similar?